debian/cloudstack-marvin.postinst

```diff
@@ -19,5 +19,6 @@
 
 set -e
 
-pip install --upgrade http://cdn.mysql.com/Downloads/Connector-Python/mysql-connector-python-2.0.4.zip#md5=3df394d89300db95163f17c843ef49df
-pip install --upgrade /usr/share/cloudstack-marvin/Marvin-*.tar.gz
+python3 -m pip install --upgrade pip
+python3 -m pip install --upgrade http://cdn.mysql.com/Downloads/Connector-Python/mysql-connector-python-2.0.4.zip#md5=3df394d89300db95163f17c843ef49df
```

Comment:
@GabrielBrascher line 23 is not needed, as pip3 install latest mysql-connector-python 8.0.15
```
root@mgt01:~# pip3 list |grep mysql-connector-python
mysql-connector-python 8.0.15
```

Reply:
Thanks for the hint, @weizhouapache. The code has been updated.